lib/rules/heuristic/group.js

```diff
@@ -4,16 +4,28 @@
 exports.name = 'heuristic.group';
 
 exports.check = function (sr, done) {
-    var patterns = /.+ interest group$|.+ community group$|.+ working group$/i;
-    var candidates = [];
-    var item;
-    var i;
+    var patterns = /.+ interest group$|.+ community group$|.+ working group$/i
+    ,   candidates = []
+    ,   candidate
+    ,   item
+    ,   i;
```

Comment:
This is such an ugly, error-prone syntax! (I am not blaming you but the very first guy who used it :) )

FYI, I plan to switch them back all to the traditional `var [...];` one. This horror has to stop and the world needs a leader :D